packages/components/components/elvis-autocomplete/src/react/autocomplete-overlay/autocompleteOverlay.tsx

```diff
@@ -39,6 +39,7 @@ export const AutocompleteOverlay = forwardRef<HTMLDivElement, AutocompleteOverla
         <AutocompletePopup
           $size={size}
```

Comment:
Why do Transient props still work? I thought they were Styled Components specific. 🤔

Reply:
I think there's a lot in common between styled components and @emotion, and perhaps this is one of the things 🤷